relayer/relays/beacon/config/config.go

```diff
@@ -56,10 +58,20 @@ type ActiveSpec string
 const (
 	Mainnet ActiveSpec = "mainnet"
 	Minimal ActiveSpec = "minimal"
+	GOERLI  ActiveSpec = "goerli"
```

Comment:
Nit: Should this not be `Goerli`? As far as I know Go constants' convention is still camel case.